src/components/ChangePlan/ChangePlan.js

```diff
@@ -178,7 +178,7 @@ export const CardWithPrice = ({ path, showFeatures, currentPlanType, promoCode }
       <CardPrice doNotShowSince={path.current} currency="US$">
         {thousandSeparatorNumber(intl.defaultLocale, path.minimumFee)}
       </CardPrice>
-      {path.current && !path.deadEnd ? (
```

Comment:
Do we use `deadEnd` property for anyting else?

Reply:
I'll use this property in my next ticket, when I have to detect which is the highest plan in the list. In case this property is true, I will show the new UI for that case. 😉

Reply:
I was deeply analizing it and It's really not necessary to mantein `deadEnd `property. I have the user plan session id, so I send it to BannerUpgrade component as a property, and It lets me compare values and check if there is just one plan in the list and if it's the current session plan. So, I should remove `deadEnd` property, are you agree? @cbernat  @lucianaDilizio

Reply:
@silsanchez8 Then in your next PR you could refactor what you need into the new UI in plan calculator. And remove what is not being used anymore.